static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -20,22 +30,52 @@ export const acquisitionsBannerFivTemplate = (
     </div>
     <div class="fiv-banner__container">
         <div class="fiv-banner__text">
-            This is a test banner for FIV - if you see it please email userhelp@theguardian.com
+        ${
+            window.location.hash.match(/[#&]fiv-potential(&.*)?$/) ||
+            (shouldSeeReaderRevenue &&
+                !window.location.hash.match(/[#&]fiv-existing(&.*)?$/))
+                ? `<div class="fiv-banner__headline fiv-banner__wide">
+                ${potentialTitleWide.markup}
+            </div><div class="fiv-banner__headline fiv-banner__mobile">
+                ${potentialTitleMobile.markup}
+            </div>
+            <div class="fiv-banner__text_bold">
+                …we remain editorially independent, our journalism free from commercial bias and our reporting open and accessible to all.
+            </div>
+            <div class="fiv-banner__text_normal">
+                Imagine what we could continue to achieve with the support of many more of you. Together we can be a force for change.
+            </div>`
+                : `<div class="fiv-banner__headline">
+                ${existingTitle.markup}
+            </div>
+            <div class="fiv-banner__text_bold">
+                …to the 1,000,000 readers who have helped to keep The Guardian’s independent journalism open and accessible to all.
+            </div>
+            <div class="fiv-banner__text_normal">
+                Imagine what the next million could achieve. Please read this update from our editor-in-chief and continue supporting us.
+            </div>`
+        }
         </div>
-        <div class="fiv-banner__cta">
-            <button class="button fiv-banner__button" href="${linkUrl}">
-                Go${arrowWhiteRight.markup}
-            </button>
-            <img
-                class="fiv-banner__payment-logos"
-                src="${config.get(
-                    'images.acquisitions.paypal-and-credit-card',
-                    ''
-                )}"
-                alt="PayPal and credit card"
-            >
+        <div class="fiv-banner__onwards">
+            <a class="fiv-banner__onwards" href="${linkUrl}">
+                Read our story${arrowRight.markup}
+            </a>
         </div>
     </div>
+    <div class="fiv-banner__cta">
+        <button class="button fiv-banner__button fiv-banner__button_contribute" href="${linkUrl}">
+            Contribute${arrowRight.markup}
+        </button>
+        ${
+            window.location.hash.match(/[#&]fiv-gb(&.*)?$/) ||
+            (geolocationGetSync() === 'GB' &&
+                !window.location.hash.match(/[#&]fiv-row(&.*)?$/))
+                ? `<button class="button fiv-banner__button fiv-banner__button_subscribe" href="${linkUrl}">
```

Comment:
this has to be `<a>` element, not a `<button>`

Reply:
OK done this and removed the block link that overlays everything.  We still need the Read our story link though.